scripts/.bump-version.js

```diff
@@ -0,0 +1,72 @@
+const replace = require("replace");
+const yaml = require('js-yaml');
+const fs = require('fs');
+const {execSync} = require('child_process')
+
+const versionRegex = /\d+\.\d+\.\d+(-\w+)?/i;
```

Comment:
can this `versionRegex` _also_ be used in other places, e.g., L32 at least?  maybe the regex at L32 can be be constructed via concatenation or string interpolation or something?

Reply:
in theory could be possible, but L6 regex is not a string is literally a regex type, and L32 regex is an string that gets converted into a regex by the replace package, and also they are 2 different regex, in practice, they are very similar and string concatenation could indeed work, but I prefer to have them as separate for ease of readability and peace of mind.
if you have a strong opinion on this I might try and make it work as suggested.